js/components/calendar.vue

```diff
@@ -48,6 +48,7 @@
 import moment from 'moment';
 
 const VIEWS = ['days', 'months', 'years'];
+const MONTH_FORMAT = 'MMMM YYYY';
```

Comment:
Yes, but in this this case this very specific to the way its used/displayed into the widget